File: gmall/gmall-web/src/main/java/com/hiscat/gmallweb/entity/OrderInfo.java
package com.hiscat.gmallweb.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("GMALL2020_ORDER_INFO")
public class OrderInfo {
    @TableField
    private String province_id;
    @TableField
    private String consignee;
    @TableField
    private String order_comment;
    @TableField
    private String consignee_tel;
    @TableField
    private String order_status;
    @TableField
    private String payment_way;
    @TableField
    private String user_id;
    @TableField
    private String img_url;
    @TableField
    private Double total_amount;
    @TableField
    private String expire_time;
    @TableField
    private String delivery_address;
    @TableField
    private String create_time;
    @TableField
    private String operate_time;
    @TableField
    private String tracking_no;
    @TableField
    private String parent_order_id;
    @TableField
    private String out_trade_no;
    @TableField
    private String trade_body;
    @TableField
    private String create_date;
    @TableField
    private String create_hour;
}

File: gmall/gmall-realtime/src/main/scala/com/hiscat/gmall/realtime/bean/OrderInfo.scala
package com.hiscat.gmall.realtime.bean

import scala.beans.BeanProperty

case class OrderInfo(
                      @BeanProperty    id: String,
                      @BeanProperty    province_id: String,
                      @BeanProperty    consignee: String,
                      @BeanProperty    order_comment: String,
                      @BeanProperty    var consignee_tel: String,
                      @BeanProperty    order_status: String,
                      @BeanProperty    payment_way: String,
                      @BeanProperty    user_id: String,
                      @BeanProperty    img_url: String,
                      @BeanProperty    total_amount: Double,
                      @BeanProperty    expire_time: String,
                      @BeanProperty    delivery_address: String,
                      @BeanProperty    create_time: String,
                      @BeanProperty    operate_time: String,
                      @BeanProperty    tracking_no: String,
                      @BeanProperty    parent_order_id: String,
                      @BeanProperty    out_trade_no: String,
                      @BeanProperty    trade_body: String,
                      @BeanProperty    var create_date: String,
                      @BeanProperty    var create_hour: String)